### DISCUSSION TITLE
Upgrade from 2.3.2 to 2.3.40 completely failed

### DISCUSSION THREAD

**Discussion by pboosten** [Security-Onion-Solutions/securityonion#3726] [Unsupported Versions]
Hi all,

Maybe I messed up, but I haven't got a clue how to resolve:

Steps taken:
- updated OS with yum install (and rebooted afterwards)
- ran soup - this ran for quite some time
- noticed that all services, except docker and so-dockerregistry were 'STARTING', then 'WAIT_START' and after that 'MISSING'.
- 'docker image ls' shows images
- so-docker-refresh -> still same result
- 'docker logs so-dockerregistry' shows this error for all images: 'msg="response completed with error" err.code="manifest unknown" err.detail="unknown tag=2.3.40"'
- removed all docker images and ran so-docker-refresh again, still no luck.

HELP! :)

Thanks in advance,
Peter

**Comment by edwardrixon** [comment]
I have had a similar issue. I fixed mine by logging onto the cli of the security onion box and navigating to the home directory of the user i specified on first install (in my case cyber). I then did a git clone of the current setup repository which pulled down a folder named securityonion.

- Next I removed all files within the SecurityOnion folder and copied all the git cloned files to the SecurityOnion Folder.
- Then I navigated to SecurityOnion/sosetup and ran the following `so-setup iso`
- This ran through the setup install again and I re-inputted my information
- Once complete I rebooted and checked the status using so-status and still had some docker container's down so I then ran so-docker-refresh and it all came back up and is working again.

All in all this took about two hours with my internet connection.

Hope this helps.

**Comment by TOoSmOotH** [comment]
What does the /root/soup.log say?

**Comment by pboosten** [comment]
> Op 2 apr. 2021, om 00:34 heeft Mike Reeves ***@***.***> het volgende geschreven:
> 
> 
> What does the /root/soup.log say?
> 

Hmm, didn’t think of looking at that. For all docker images this is the result:

Updating dockers to 2.3.40.
Downloading so-acng:2.3.40
No valid trust data for 2.3.40
Error response from daemon: No such image: securityonion/so-acng:2.3.40
The push refers to repository [securityonion:5000/securityonion/so-acng]
tag does not exist: securityonion:5000/securityonion/so-acng:2.3.40
